data/tutorial/introduction.md

```diff
@@ -7,4 +7,4 @@ title: "Introduction"
 
 Typically Irmin is accessed by embedding it into an OCaml application, but can also be accessed using HTTP using `irmin-http` or GraphQL using `irmin-graphql`. It is most often used to store application data, like configuration values, shared state or checkpoint data, but there is nothing stopping you from using it as a general purpose key-value store too. Additionally, since it is compatible with Git, Irmin can be used to interact with Git repositories directly from within your application.
 
-Take a moment to skim the [README](https://github.com/mirage/irmin/blob/master/README.md) to familiarize yourself with some of the concepts. Also, if you find that anything is missing or unclear in this tutorial then please file [an issue](https://github.com/zshipko/irmin-tutorial/issues)!
+Take a moment to skim the [README](https://github.com/mirage/irmin/blob/master/README.md) to familiarize yourself with some of the concepts. Also, if you find that anything is missing or unclear in this tutorial then please file [an issue](https://github.com/tarides/irmin.io/issues)!
```

Comment:
Unrelated to this PR, but perhaps we should be hardcoding a "Submit Issue" link somewhere in the footer of each tutorial page, rather than just in the intro.